concepts.wip/booleans/introduction.md

```diff
@@ -2,3 +2,37 @@
 
 True or false values are represented by the `Bool` type.
 It contains only two values: `true` and `false`.
+
+In Julia, it is a built-in _data_ _type_.
+
+> In programming languages with a built-in Boolean data type, such as Pascal and Java[^1], the comparison operators such as `>` and `!=` are usually defined to return a Boolean value[^2].
+
+[^1]: This also includes Julia, among other programming languages.
+[^2]: [Programming generalities](https://en.wikipedia.org/wiki/Boolean_data_type#Generalities).
+
+## Booleans in Julia
+
+```julia
+julia> true
+true
+
+julia> false
+false
+
+# We can check for types in Julia,
+# with the `typeof` function.
+julia> typeof(true)
+Bool
+
+julia> typeof(false)
+Bool
+
+# Another way to directly "typecheck".
+# isa(x, type) -> Bool
+julia> isa(true, Bool)
+true
+
+# "isa" function can be used as an infix operator.
+julia> false isa Bool
+true
+```
```

Comment:
I think this would fit better into a general concept of types, and what has a type in Julia etc. rather than the introduction to Bools. Perhaps you could move it there?

Reply:
Yes, I'll add it there, when writing that section. Thanks. 

I'll remove it in the Booleans section.